scripts/sed

```diff
@@ -65,7 +65,7 @@ s:\<Key/value:Key-value:g
 
 s/[^#"]\<data-?type/data type/g
 
-s/([^\./`-])[Jj]son/\1JSON/
+s/([^\./`-])[J]son/\1JSON/
```

Comment:
Better to change from any spelling of jSoN to json; and any spelling of JsOn to JSON?

Reply:
I agree, while these types of spelling will probably be rare, it is better if we make sure to fix them.

Reply:
You never know where a typo will happen :wink: 

Btw. character class with only one character is not needed.